src/components/News/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import {useState} from "react";
+import styled from 'styled-components'
+import fuse from '../../assets/svg/fuse_sub.svg'
+import { ChevronUp } from 'react-feather'
+import Card from '../MainCard'
+
+export const Header = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto;
+  width: 100%;
+  background: ${({ theme }) => theme.bg1};
+  border-bottom: solid 2px #E4DDDD;
+  line-height: 0rem;
+`
+export const HeaderItem = styled.div`
+  display: flex;
+  flex: 1 1 90%;
+  flex-wrap: wrap;
+  padding-bottom: 1rem;
+`
+const HeaderOptions = styled("div")<{ size: string, y: string }>`
+  display: flex;
+  flex: 1 1 10%;
+  flex-wrap: wrap;
+  transform: rotateX(${({ size }) => size}deg) translateX(10px) translateY(${({ y }) => y}px);
+  transition: all 1s;
+`
+
+const ExpandableWrapper = styled.div`
+overflow: hidden;
+`
+
+const Content = styled("div")<{ size: string }>`
+margin-top: ${({ size }) => size}%;
+transition: all 0.4s;`
+
+export default function NewsModal() {
+  const [open,setOpen] = useState(false);
+  const articles = [
```

Comment:
I suggest to store the articles in a dedicated json file so it can be updated without touching code